src/migrationTransformer/migrationTransformer.ts

```diff
@@ -0,0 +1,20 @@
+import { PrismaTemplates } from '../'
+import migrations from '../generatedMigrations/index'
+import { BaseTemplateParametersResolved } from '../types'
+
+export type MigrationSql = string[]
+
+export type Params = {
+  template: PrismaTemplates.$Types.TemplateTag
+  parameters: BaseTemplateParametersResolved
+}
+
+export const selectSql = (params: Params): MigrationSql => {
+  if (
+    params.parameters.datasourceProvider === 'sqlite' ||
```

Comment:
SQLite does not have migrations?

Reply:
We don't support sqlite, but I can add it anyways if `migrate diff` permits?

Reply:
Yeah, let's keep the cutoff on this as late in the flow as possible. Would prefer to not make that decision at this level.